src/Silex/ServiceProviderInterface.php

```diff
@@ -23,8 +23,6 @@
      *
      * This method should only be used to configure services and parameters.
      * It should not get services.
-     *
-     * @param Application $app An Application instance
```

Comment:
Isn' the phpdoc invalid if it contains information, but not the parameters used? I am not saying the "An application instance" should stay. Just the `@param` tag.